src/query/functions/src/aggregates/aggregate_array_agg.rs

```diff
@@ -417,6 +418,7 @@ pub fn try_create_aggregate_array_agg_function(
     display_name: &str,
     _params: Vec<Scalar>,
     argument_types: Vec<DataType>,
+    _sort_descs: Vec<AggregateFunctionSortDesc>,
```

Comment:
We can also implement within group for `array_agg` function, which is similar to `string_agg` function.